governance/docs/what_is_governance.md

```diff
@@ -0,0 +1,61 @@
+# What Is Governance?
+
+Because open source projects are organizations, all projects have governance. Some of governance structures are more *explicit* than others. Some are more *formal* than others. But every project has them.
```

Comment:
Every project where a community forms has some types of norms and expectations for how decisions are made and how conflicts are resolved. But not all open-source software is released with the intention of enabling collaborative development. Some open-source software is still released as snapshots without, for example, the Git commit history between releases.

Reply:
No project in the CNCF is this way.

Reply:
Then say "all CNCF projects have governance"